ocp-templates/app-interface/scheduler.yaml

```diff
@@ -0,0 +1,136 @@
+# Template for Ibutsu scheduler in AppInterface
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ibutsu-scheduler-template
+# ===============================================
+# Scheduler
+# ===============================================
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-scheduler
+  spec:
+    replicas: 1
+    selector:
+      deploymentConfig: ibutsu-scheduler
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: ${APP_NAME}
+          deploymentConfig: ibutsu-scheduler
+      spec:
+        containers:
+        - env:
+          - name: POSTGRESQL_HOST
+            valueFrom:
+              secretKeyRef:
+                key: db.host
+                name: ibutsu-db
+          - name: POSTGRESQL_PORT
+            valueFrom:
+              secretKeyRef:
+                key: db.port
+                name: ibutsu-db
+          - name: POSTGRESQL_USER
+            valueFrom:
+              secretKeyRef:
+                key: db.user
+                name: ibutsu-db
+          - name: POSTGRESQL_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: db.password
+                name: ibutsu-db
+          - name: POSTGRESQL_DATABASE
+            valueFrom:
+              secretKeyRef:
+                key: db.name
+                name: ibutsu-db
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: db.password
+                name: redis
+                optional: true
+          - name: REDIS_HOSTNAME
```

Comment:
Should we be fetching the redis values from a secret in a similar way in the other templates too?

Reply:
Probably, I'll fix that up with my changes for redis and rds.